src/main/java/com/thinkful/noteful/tags/TagDeserializer.java

```diff
@@ -0,0 +1,36 @@
+package com.thinkful.noteful.tags;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+
+import com.thinkful.noteful.NoteException;
+
+import java.io.IOException;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class TagDeserializer extends StdDeserializer<Tag> {
+
+  @Autowired
+  TagRepository tagRepository;
+
+  public TagDeserializer() {
+    this(null);
+  }
+
+  public TagDeserializer(Class<?> vc) {
+    super(vc);
+  }
+
+  @Override
+  public Tag deserialize(JsonParser jp, DeserializationContext ctx) 
+        throws IOException, JsonProcessingException {
```

Comment:
`JsonProcessingException` is a subclass of `IOException` so we can drop it without weakening our error-handling here.

Reply:
@vramdhanie - Sure, it's unneeded from a programmatic standpoint but does help to illustrate using the newer multi-catch syntax.